Add doc to custom URScript commands that it needs to be in headless mode (#1051)
UniversalRobots/Universal_Robots_ROS2_Driver: ur_robot_driver/doc/usage.rst

------------------------

The driver's package contains a ``urscript_interface`` node that allows sending URScript snippets

> directly to the robot when the robot is in headless/Remote control mode.

Reviewer: Wether or not we use headless mode doesn't really matter. But yes, we should state more clearly, that the robot will have to be in remote control mode for a PS5 robot.

```suggestion
directly to the robot when the robot is in remote control mode.
```

Author: True it the remote control that makes it accepting code from external on Polyscope5

external_control program again.

.. note::

>   Custom URScript command only works in headless mode. And does not work with the external_control URCap.

Reviewer: I think, this  is  a bit misleading, as in fact it is working together with the external_control URCap. The program just has to be restarted afterwards, as stated above.

Author: @fmauch No that is not correct. On the e-series it only will work when the robot in remote control.
Maybe that is what we should write

>   Custom URScript command only works in headless mode. And does not work with the external_control URCap.

Author: ```suggestion
  Custom URScript command only works in remote control mode on E-series robots or later.
```

Reviewer: How about: "On E-series robots or later the robot needs to be in remote control mode in order to execute custom URScript commands."

Reviewer 2: > "On E-series robots or *newer* the robot needs to be in remote control mode in order to execute custom URScript commands."

Author: ```suggestion
  On E-series robots or newer the robot needs to be in remote control mode in order to execute custom URScript commands.
```
